lib/v2/sanlian/tag.js

```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const util = require('@/v2/nintendo/utils');
+
+const rootUrl = 'https://www.lifeweek.com.cn/articleList';
+const articleRootUrl = 'https://www.lifeweek.com.cn/article';
+module.exports = async (ctx) => {
+    const tag = ctx.params.id;
+    const url = `${rootUrl}/${tag}`;
+    const response = await got(url, {
+        responseType: 'buffer',
+    });
+    const result = await util.nuxtReader(response.data);
+    if (!result.articleList) {
+        throw new Error('新闻信息不存在，请报告这个问题');
+    }
```

Comment:
You can use the API endpoint of `/api/userWebFollow/getFollowTagContentList` instead.

Reply:
<img width="593" alt="image" src="https://github.com/DIYgod/RSSHub/assets/105254603/8f4e6827-a01b-469e-a127-0c8bd23b01be">

I can not find this api in Chrome Dev Console, and the name of the api indicates that it will provide user-specific content, which is not desirable for non-login viewer.

Reply:
Have you tried switching sorted by date/trend?

Reply:
Do you mean `Sort by Time`?
<img width="303" alt="image" src="https://github.com/DIYgod/RSSHub/assets/105254603/e758689a-22b7-480c-af2c-20f570251a87">

Network requests/responses are sorted, but API endpoint `/api/userWebFollow/getFollowTagContentList` is still missing. What is wrong about my process? Could you please help me? Thanks a lot1

Reply:
<details><summary>Screenshot</summary>

![image](https://github.com/DIYgod/RSSHub/assets/11386903/d7e648ba-ddaa-4771-a50c-522b1f38481b)
</details>

Reply:
Thanks! I get it